charts/brokencrystals/templates/_helpers.tpl

```diff
@@ -27,7 +27,7 @@ If release name contains chart name it will be used as a full name.
 Create chart name and version as used by the chart label.
 */}}
 {{- define "brokencrystals.chart" -}}
-{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 50 | trimSuffix "-" }}
```

Comment:
Why was it changed?

Reply:
I changed 63 to 50 everywhere, thinking that this would reduce the likelihood of errors due to the length of the name. Later I changed it back, it seems these parts are not actively involved.